src/helpers/toggleMusic.ts

```diff
@@ -0,0 +1,16 @@
+import { Scene } from 'phaser';
+
+const toggleMusic = (scene: Scene) => {
+  const btn = document.getElementById('music');
+  const el = document.getElementById('game');
+  if (btn && el) {
+    btn.addEventListener('click', () => {
+      const isMute = scene.game.sound.mute;
+      // eslint-disable-next-line no-param-reassign
+      scene.game.sound.mute = !isMute;
```

Comment:
i'm wondering if we should disable this linting rule because i think i've also run into this issue and it seems like recommended way of using some of phaser

Reply:
the eslint overrides to rules are defined here

https://github.com/ScottyXIII/bigheaded/blob/main/.eslintrc.cjs#L21-L24

and here is the man page for `no-param-reassign`

https://eslint.org/docs/latest/rules/no-param-reassign#options

Reply:
Yeah could do if it's gonna cause issues a lot! I'll see if I can add it to this PR